api/rootcerts.go

```diff
@@ -26,6 +26,10 @@ type certConfig struct {
 
 	// CAPath is a path to a directory populated with PEM-encoded certificates.
 	CAPath string
+
+	// SystemCerts determines if system certificates should be included in the pool.
+	// Defaults to true
+	SystemCerts *bool
```

Comment:
```suggestion
	SkipSystemCerts bool
```

I think we get the same semantics if we invert and simplify the logic. This saves callers from having to deal with the allocation here which is sometimes yucky. 

My 2c, but it really is a nitpick at this point.

Reply:
Aah I thought they would be a better way... I updated the logic :)